pages/linux/strip.md

```diff
@@ -0,0 +1,16 @@
+# strip
+
+> Discard symbols from executables or object files.
+> More information: <https://manned.org/strip>.
+
+- Replace the input file with its stripped version:
+
+`strip {{path/to/file}}`
+
+- Put the stripped output in file, rather than replacing the existing one:
```

Comment:
I think this is more explicit and cleaner:
```suggestion
- Strip symbols from a file, saving the output to a specific file:
```

Reply:
agree